### DISCUSSION TITLE
users table needs to get published manually

### DISCUSSION THREAD

**Discussion by KnudH** [Budibase/budibase#7649] [Help] [answered]
I clicked my first app and it's working nice so far!
There is only one thing I'm curious about and can't find anything about in the docs.

If I update the build in users-table, this update don't get applied to the app, until I click “publish” again. 

Is this the expected behavior? I think this should be in the docs for sure!

**Comment by andz-bb** [comment, marked as answer]
Hey Knud, glad to hear your first app is working well! 😊

When you update the Users table in the builder it is only updating it in the dev DB and whenever you hit publish the changes then sync from that to the production DB. Sorry that it wasn't clear to you! We will update the docs soon to include info on this.
